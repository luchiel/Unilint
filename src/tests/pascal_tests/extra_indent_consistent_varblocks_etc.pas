type
  a = int;
  
procedure abaca;
  const
    MILLE: int = 1000;
  var
    i: integer;
  begin
    do_nothing();
  end;
  
var
  freedom: char;
begin
  for freedom := 0 to 10 do begin
      //error next line :)
  end;
end.