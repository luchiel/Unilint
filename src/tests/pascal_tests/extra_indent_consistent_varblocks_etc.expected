18:1: indent too small
